vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,195 @@
+
+Based on this, we've shipped some of the following features to make it easier for these users to get started with Dendron:
+
+- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/): This was a complete redesign of our getting started tutorial. We've added links to all concepts as well as step-by-step instructions on each option (instead of assuming prior knowledge on the user).
+- Improvements to `Change Workspace` and `Initialize Workspace` now make navigation easier for users to select workspaces via a file explorer.
+- [[The Five Minute Journal with Dendron and Visual Studio Code|blog.2022.2022-01-05-five-minute-journal-dendron-vscode]]: This blog post was made due to interest in [schemas](https://wiki.dendron.so/notes/c5e5adde-5459-409b-b34d-a0d75cbb1052/) and [templates](https://wiki.dendron.so/notes/861cbdf8-102e-4633-9933-1f3d74df53d2/). We created the blog post and updated the documentation to create a better introduction to the power of these features.
+- Users are now prompted when running [`Delete Node`](https://wiki.dendron.so/notes/eea2b078-1acc-4071-a14e-18299fc28f47/#delete-node), and shown what links will break throughout documentation _before_ confirming deletion.
+  - [`Dendron Doctor`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/) has also had similar improvements, such as [`findBrokenLinks`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/#findbrokenlinks), to make it easier to see where documentation improvements can be made.
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management. For users coming in with prior experience, there was no single tool that folks were migrating from. Instead, we had a massive list of previous tools. For those coming from existing tools, we have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that continues to grow.
+
+> We are looking to the community to [help us improve and expand the source documentation](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md)!
+
+### Interoperability
+
+Within the Dendron Discord, users have mentioned their interest in _interoperability_: they want a single source that they export/import their notes to/from. Different tools co-exist in workflows and can be used concurrently.
+
+Our users continue to integrate information managed by Dendron with **pods** or their own tooling. In Dendron, [pods](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f/) refer to any tool or location that contains information and how Dendron manages that information: [Markdown files](https://wiki.dendron.so/notes/13c4a608-0a32-4c62-9c7e-2b7f9f2d18bf/), [GitHub issues](https://wiki.dendron.so/notes/PuDUEyUAPmvpBvaFWHDOS/), [Notion](https://wiki.dendron.so/notes/eOw3TC9KgdbBvhyU7pyGl/), and social media (Twitter, Facebook, etc.) are pod examples.
```

Comment:
This description is confusing. Pods are a set of plugins that help users synchronize their notes between Dendron and other tools